src/views/Graph.vue

```diff
@@ -558,10 +584,13 @@ export default {
       this.updating = true
 
       // extract the graph (non reactive lists of nodes & edges)
-      const nodes = this.getGraphNodes()
+      const nodes = await this.waitFor(() => {
+        const nodes = this.getGraphNodes()
+        return nodes.length ? nodes : false
```

Comment:
```suggestion
        return this.getGraphNodes()
```

Reply:
This causes the autorefresh issue to return
![image](https://github.com/cylc/cylc-ui/assets/7523921/9410287e-c326-4171-b75f-b77f2eed8e16)

Reply:
Ah my bad